gst/nnstreamer/tensor_converter/tensor_converter.c

```diff
@@ -463,7 +462,9 @@ gst_tensor_converter_sink_event (GstPad * pad, GstObject * parent,
       silent_debug_caps (in_caps, "in-caps");
 
       if (gst_tensor_converter_parse_caps (self, in_caps)) {
-        out_caps = gst_tensor_caps_from_config (&self->tensor_config);
+        GstTensorConfig tensor_config;
+        gst_tensor_config_from_tensors (&tensor_config, &self->tensors_config);
```

Comment:
out_caps maybe other/tensor(s). plz check below.
```
out_caps = gst_tensors_caps_from_config (&self->tensors_config);
if (num-tensors == 1) {
   config single-tensor
  gst_caps_append (out_caps, gst_tensor_caps_from_config (&tensor_config));
}
```

Reply:
Except for the sub-plugin, the tensor is supported only. 
And sub-plugin updates cap after checking the number of tensor and peer_caps in chain function.

Reply:
Modified to support tensor/tensors. Thank you for your valuable comments.